src/abstractdataframe/show.jl

```diff
@@ -405,14 +405,28 @@ function showrows(io::IO,
     end
 
     rowmaxwidth = maxwidths[ncols + 1]
+    colwidths = maxwidths .+ length(" ") .+ length(" |")  # add border + padding
```

Comment:
probably it is cleaner to use `textwidth` not `length` (but in this case it is the same - it is just a matter of consistency in the codes).

Reply:
Cool, I didn't know about `textwidth`. I'll switch over to using it.